### PR TITLE
sm7150: surya: Add ACPI table

### PR DESCRIPTION
**Pull request by Icesito68**
With this the x3 nfc starts windowspe and windows 11 but without usbç

### Description

[Description of the changes you are submitting]

### Checklist

* [ x] Have you tested the change you are submitting?
* [ x] Is the commits history nice and clean?

### CODE REVIEW

**Comment by MollySophia** [comment]
Thanks a lot for your work!
Well, could you please squash some of your comits into several key ones? That would be better

File: Platform/Xiaomi/sm7150/surya.fdf.inc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+FILE DRIVER = 8e9bd160-b184-11df-94e2-0800200c9a66 {
+    SECTION DXE_DEPEX = Platform/EFI_Binaries/Drivers/sm7150/DALSys/DALSys.depex
+    SECTION PE32 = Platform/EFI_Binaries/Drivers/Devices/surya/DALSys/DALSys.efi
+    SECTION UI = "DALSys"
+}
+
+FILE DRIVER = 5bd181db-0487-4f1a-ae73-820e165611b3 {
+    SECTION DXE_DEPEX = Platform/EFI_Binaries/Drivers/sm7150/ButtonsDxe/ButtonsDxe.depex
+   SECTION PE32 = Platform/EFI_Binaries/Drivers/Devices/surya/ButtonsDxe/ButtonsDxe.efi
```
Comment (ungeskriptet) [inline Platform/Xiaomi/sm7150/surya.fdf.inc:11]: Unneeded, ButtonsDxe is already patched.

**Review by qaz6750 — CHANGES_REQUESTED** [review]
Please use git to manage submission correctly

File: Platform/Xiaomi/sm7150/surya.fdf.inc (hunk 1 above)
Comment (Icesito68) [reply Platform/Xiaomi/sm7150/surya.fdf.inc:11]: sorry jeje

**Pull request closed without merging**